homework/index.js

```diff
@@ -30,14 +32,145 @@
     return elem;
   }
 
+  function openNewTab(url) {
+    window.open(url, '_blank');
+  }
+
+  function bindContributions(contributorsUrl) {
+    fetchJSON(contributorsUrl, (error, response) => {
+      const repoInfoContainer = document.getElementById('repo-info-container');
+      const contributorsDiv = createAndAppend('div', repoInfoContainer, {
+        class: 'contributors-info',
+      });
+      if (error) {
+        createAndAppend('div', contributorsDiv, {
+          text: error.message,
+          class: 'alert-error',
+        });
+        return;
+      }
+
+      const div = createAndAppend('div', contributorsDiv);
+      createAndAppend('h4', div, { text: 'Contributions' });
+      createAndAppend('hr', div);
+      const table = createAndAppend('div', div, { class: 'grid' });
+      const contributors = response;
+      for (let index = 0; index < contributors.length; index++) {
+        const imageUrl = contributors[index].avatar_url;
+        const name = contributors[index].login;
+        const contributionsCount = contributors[index].contributions;
+        const profileURL = contributors[index].html_url;
+        const tr = createAndAppend('div', table, { class: 'clickable row' });
+        tr.onclick = () => openNewTab(profileURL);
+        const td1 = createAndAppend('div', tr, { class: 'col-30' });
+        const imgContainerDiv = createAndAppend('div', td1, {
+          class: 'image-container',
+        });
+        createAndAppend('img', imgContainerDiv, {
+          src: imageUrl,
+          width: 80,
+          height: 80,
+        });
+        createAndAppend('div', tr, { text: name, class: 'col-30 description' });
+        const td3 = createAndAppend('div', tr, { class: 'col-30' });
+        createAndAppend('label', td3, { text: contributionsCount });
+      }
+    });
+  }
+  function bindRepositoryDetails(selectedRepository) {
+    const { description } = selectedRepository.description;
+    const { forks } = selectedRepository.forks;
+    const repositoryName = selectedRepository.name;
+    const updatedOn = new Date(selectedRepository.updated_at).toLocaleString();
+    const repoURL = selectedRepository.html_url;
+
+    const infoContainerDiv = document.getElementById('repo-info-container');
+    const repoInfodiv = createAndAppend('div', infoContainerDiv, {
```

Comment:
You can avoid a spelling error being flagged here by using camelCase also for the `div` part: `repoInfoDiv`.

Reply:
done